apps/app/components/CodeBlock/index.tsx

```diff
@@ -39,7 +58,10 @@ const CodeBlock: FunctionComponent<CodeBlockProps> = ({ children, hidden, event
   const { theme = "light" } = useTheme();
   hljs.registerLanguage("python", python);
   hljs.registerLanguage("javascript", javascript);
-  hljs.registerLanguage("html", html_xml);
```

Comment:
Same here

Reply:
My mistake - removed it because I thought I added it in accidentally
Rectified here: https://github.com/data-gov-my/datagovmy-front/pull/175/commits/58e4687e2f83914c009c8f80fe19b13c6ddc3cc8